tests/apollo/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ if(THREADCHECK)
   # For Leak Sanitizers to report errors we need to shut down processes gracefully
   set(APOLLO_TEST_ENV "${APOLLO_TEST_ENV} GRACEFUL_SHUTDOWN=true")
   set(APOLLO_TEST_ENV "${APOLLO_TEST_ENV} \
-  TSAN_OPTIONS=log_path=${TSAN_LOG_DIR}/leak.log:history_size=7 \
+  TSAN_OPTIONS=log_path=${TSAN_LOG_DIR}/tsan.log:history_size=7 \
```

Comment:
I suggest having the full path of the log file as an environment variable.

Reply:
Not sure I understand. This ist he default full path (e.g /concord-bft/build/tsan_logs)

Reply:
new task:
https://jira.eng.vmware.com/browse/BC-26378